test/unit/component/registry/bpmn-model-registry.test.ts

```diff
@@ -62,6 +81,47 @@ describe('Bpmn Model registry', () => {
     });
   });
 
+  it('search intermediate catch event', () => {
+    bpmnModelRegistry.load(
+      intermediateCatchInModel('intermediate catch event id', 'intermediate catch event name', { incomingIds: ['incoming_1'], outgoingIds: ['outgoing_1', 'outgoing_2'] }, [
```

Comment:
**suggestion:** having a `intermediateCatchEventInModel` function would be clearer.

Reply:
I forgot the 'Event' when I renamed 😆